pkg/deployment/deployment.go

```diff
@@ -315,6 +315,34 @@ func (builder *Builder) WithServiceAccountName(serviceAccountName string) *Build
 	return builder
 }
 
+// WithVolumes sets the Volume on the deployment definition.
+func (builder *Builder) WithVolumes(volume *coreV1.Volume) *Builder {
```

Comment:
There is only 1 volume in argument. Please re-name the function to **WithVolume**

Reply:
the final parameter name is volumes, but as you wish